apps/examples/testcase/le_tc/kernel/tc_timer.c

```diff
@@ -44,15 +44,37 @@ static void tc_timer_timer_create_delete(void)
 	int ret_chk = 0;
 	timer_t timer_id;
 	timer_t gtimer_id;
-	clockid_t clockid = CLOCK_REALTIME;
+	clockid_t clockid = CLOCK_MONOTONIC;
 	struct sigevent st_sigevent;
 	struct posix_timer_s *st_ret_prt;
 	struct itimerspec st_timer_spec_val;
+
 	/* Set and enable alarm */
+
 	st_sigevent.sigev_notify = SIGEV_SIGNAL;
 	st_sigevent.sigev_signo = sig_no;
 	st_sigevent.sigev_value.sival_ptr = &timer_id;
 
+	/* Check for NULL clock ID != CLOCK_REALTIME */
```

Comment:
check comments, Is it check for ` NULL clock ID ` or `clock ID != CLOCK_REALTIME `case ??